CONTRIBUTING.md

```diff
@@ -1,108 +1,136 @@
-2. **Reviewing pull requests for the docs.** Reviewing [documentation updates](https://github.com/facebook/react-native-website/pulls) can be as simple as checking for spelling and grammar. If you encounter situations that can be explained better in the docs, click **Edit** at the top of most docs pages to get started with your own contribution.
-3. **Help people write test plans.** Some pull requests sent to the main repository may lack a proper test plan. These help reviewers understand how the change was tested, and can speed up the time it takes for a contribution to be accepted.
+- When you run `git remote -v`, you should see only your fork in the output list
 
-Each of these tasks is highly impactful, and maintainers will greatly appreciate your help.
+```bash
+git remote -v
 
-### Our Development Process
+     origin  https://github.com/johndoe/react-native-macos.git (fetch)
 
-We use GitHub issues and pull requests to keep track of bug reports and contributions from the community. All changes from engineers at Facebook will sync to [GitHub](https://github.com/facebook/react-native) through a bridge with Facebook's internal source control. Changes from the community are handled through GitHub pull requests. Once a change made on GitHub is approved, it will first be imported into Facebook's internal source control and tested against Facebook's codebase. Once merged at Facebook, the change will eventually sync back to GitHub as a single commit once it has passed Facebook's internal tests.
+     origin  https://github.com/johndoe/react-native-macos.git (push)
+```
 
-You can learn more about the contribution process in the following documents:
+- Map the primary **react-native-macos** repository as the upstream remote
 
-* [Issues](https://github.com/facebook/react-native/wiki/Issues)
-* [Pull Requests](https://github.com/facebook/react-native/wiki/Pull-Requests)
+```bash
+git remote add upstream https://github.com/microsoft/react-native-macos.git
+```
 
-We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).
+- Now running `git remote -v` should show the upstream repository also
 
-### Repositories
+```bash
+git remote -v
 
-The main repository, <https://github.com/facebook/react-native>, contains the React Native framework itself and it is here where we keep track of bug reports and manage pull requests.
+     origin  https://github.com/johndoe/react-native-macos.git (fetch)
```

Comment:
nit: tabs after the aliases are aligning the urls in a weird way.   Could just use spaces to manually align the urls on the same column.

Reply:
Never mind: it take it back.   The same alignment is exactly what one sees on the command line.  🤪